server/lint-teleport-docs-links.ts

```diff
@@ -24,6 +24,11 @@ const isMdxComponentWithHref = (node: Node): node is MdxAnyElement => {
 };
 
 const isAnAbsoluteDocsLink = (href: string): boolean => {
+  // The href is not a string value, and is likely an AST node object. Skip
+  // the absolute link check.
+  if (typeof href != "string") {
+    return false;
+  }
   return (
```

Comment:
if typescript is doing its job, this should only be able to be called with a string anyway. but, i suppose it doesnt hurt to be extra safe

Reply:
> if typescript is doing its job, this should only be able to be called with a string anyway. but, i suppose it doesnt hurt to be extra safe

👍 For some reason, TypeScript is letting `remark` call the link linter on non-strings, I guess because it doesn't check the site where remark calls its plugins.